tools/codegen/gen_functionalization_type.py

```diff
@@ -232,7 +250,7 @@ def emit_inplace_functionalization_body(
       {{
           at::AutoDispatchSkipFunctionalize guard;
           // The functionalization pass explicitly doesn't pass out= parameters to the redispatch
-          tmp_output = at::_ops::{functional_op.func.name.unambiguous_name()}::redispatch({', '.join(functional_exprs)});
+          {functional_call_str}
```

Comment:
Maybe we should have landed your test PR before making this change because I'm about to suggest some more test cases haha